frontend/pages/login.js

```diff
@@ -30,7 +34,19 @@ export default function Login({ setIsLogged = () => {}, setEmail = () => {} }) {
       <p className="title">Hi there 👋</p>
       <p className="description">I’ll help you to never lose your links again. To continue, simply log in.</p>
       <div className="google-login-wrapper">
-        <GoogleButton isLoading={loading} onClick={handleLogin} />
+        <Input
+          id="email-input"
+          value={userEmail}
+          onChange={(value) => setUserEmail(value.target.value)}
+          text="Email"
+        />
+        <Button
+          isLoading={loading}
+          onClick={handleAddLink}
+          text={initialValue?.id ? 'Edit link' : 'Add link'}
```

Comment:
```suggestion
          text="Login"
```
Aqui seria login né?